docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -5,86 +5,47 @@ lang: en
-
-2. Turn on Einstein for Developers.
-
-![Einstein Terms and Toggle](./images/einstein-terms.png)
-
-**Important**: We encourage you to thoroughly review the license agreement and review all terms and conditions. Then accept to enable Einstein for Developers.
-
-Einstein for Developers is activated in the org.
-
-### Add Users to the Org
-
-An admin can add additional users to the org as needed:
-
-1. From Setup, in the Quick Find box, enter `Users`, and then select **Users**.
-2. Click **New User** or **Add Multiple Users**.
-3. Select the appropriate license type and profile based on the user’s role.
-4. Select **Generate passwords and notify user via email**.
-5. Click **Save**.
-   This procedure generates an email inviting the new users into the org.
-
 ### Visual Studio Code Version
 
 VS Code releases a new version each month with new features and important bug fixes. You must be on VS Code Version 1.76 or higher to run the Einstein for Developers extension. You can manually check for updates from **Help** **> Check for Updates** on Linux and Windows or **Code > Check for Updates** on macOS.
 
-## Install Einstein for Developers Extensions
-
-Install Einstein for Developers:
-
-- If you’re using VS Code on your desktop, install [Einstein for Developers](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-einstein-gpt) from the Visual Studio Code marketplace.
-
-- If you are using Code Builder, click the Extensions icon in the Activity Bar in Code Builder, search for “Einstein for Developers” and click **Install**.
+### Interaction with other AI-enabled Extensions
 
 - You may run into some unexpected generated code outcomes if you have multiple AI-enabled extensions installed in VS Code. We recommend you disable all other AI-enabled extensions when working with Einstein for Developers:
   1. Click the Extensions icon in the Activity Bar, search for the extension to disable by name.
   2. Click **Disable** in the extension's marketplace page.
   3. Repeat for all installed AI-enabled extensions.
- 
 
-**Note**: To use Einstein for Developers on your desktop, you must have the [Salesforce Extension Pack]() installed in your VS Code desktop application. See [Install Salesforce Extensions](https://developer.salesforce.com/tools/vscode/en/vscode-desktop/install) for more information.
 
-## Connect to an Org
+## Enable or Disable Einstein For Developers in VS Code
 
-The Einstein for Developers tool is run in the context of a Salesforce org, in a Salesforce DX project. To use this tool:
+You can enable or disable Einstein for Developers from within Visual Studio Code. Click the Einstein icon in the status bar to open the command palette, then click enable or disable.
```

Comment:
Title makes it sound like it's VS Code only, can we clarify it'll apply to both VS Code and CB?